### PR TITLE
[DUOS-302][risk=no] Approval Expiration Time resource and service cleanup

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
Ongoing cleanup from instance holder to service class pattern: https://broadinstitute.atlassian.net/browse/DUOS-302

This feature is a relatively unused one that sets the default timeout value for all elections. There can only ever be a single value, the default is 7 days.

## Changes
* Added the missing api docs for the resource class
* Reformatted dao class and added tests
* Refactored the 3 instance holder classes to a single service class
* Fixes a bug in the POST that would always return a 500 error even if it succeeded.

----

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
- [ ] I've updated Swagger to reflect any API changes

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/ConsentApplication.java [hunk 1]
```diff
@@ -221,6 +220,7 @@ public void run(ConsentConfiguration config, Environment env) {
         final AssociationDAO associationDAO = injector.getProvider(AssociationDAO.class).get();
 
         // Services
+        final ApprovalExpirationTimeService approvalExpirationTimeService= injector.getProvider(ApprovalExpirationTimeService.class).get();
```
Comment (lindojong) [inline src/main/java/org/broadinstitute/consent/http/ConsentApplication.java]: nit: space

**Review by lindojong — COMMENTED** [review]
Overall, great job cleaning up! You may have missed deleting the code for `DatabaseApprovalExpirationTimeAPI` and its `DatabaseApprovalExpirationTimeAPITest`

**Comment by rushtong** [comment]
Thank you @lindojong 

> Overall, great job cleaning up! You may have missed deleting the code for DatabaseApprovalExpirationTimeAPI and its DatabaseApprovalExpirationTimeAPITest

For these, I had meant to rename the tests ... thanks for catching that! Wherever we see anything named `Database....`, we have technical debt :-)

**Comment by rushtong** [comment]
@lindojong Updated code with some cleanup and formatting. Should be no more unused code related to these changes.

**Review by lindojong — APPROVED** [review]

**Pull request merged**